pandas/guide/source/pandas_setup.rst

```diff
@@ -104,19 +104,16 @@ To fetch the latest updates from the pandas repository, follow the steps in
 
 * Create a conda environment: ::
 
-    conda env create -n pandas_dev -f <path-to-pandas-dir>/ci/environment-dev.yaml
+    conda env create
 
   .. note::
     **Windows users**: If you're copy-pasting the path, replace all pasted
     ``\`` characters with ``/`` for the command to work.
```

Comment:
Sorry, didn't notice before, this not doesn't apply anymore, as there is not path in the previous command now.

Reply:
No, that's my bad. I should have noticed that. Thanks for your patience.